File: 배열/JunYoungKr/인싸가되고싶은민수.py
a, b = map(int, input().split(' '))
print(a,b)

if (a != b): print("2")
else:
    i = 2
    while i * i <= a:
        if a % i == 0:
            print(i)
            break
        i+=1
    else:
        print(a)
        

File: 비타알고/hani0903/인싸가되고싶은민수.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? 'dev/stdin' : '../../input.txt';
const [a, b] = fs.readFileSync(filePath, 'UTF-8').trim().split(' ').map(Number());

function solution(){

    if(b === a){

        for(let i = 2; i <= b; i ++){
            if(b % i === 0){
                return i;
            }
        }
    }else{
        return 2;
    }
}

console.log(solution());
